docs/source/installation/for_dev.rst

```diff
@@ -0,0 +1,128 @@
+Manim Installation For Developers
+=================================
+
+	This documentation is for developers who want to contribute to ``ManimCommunity/manim``.
+
+Installing Poetry
+*****************
+
+Poetry can be easily installed in any OS by just running the below command.
+
+If your system has the ``curl`` program installed, which is usually true in Linux and Mac,
+.. code-block:: bash
+	
+	curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/get-poetry.py | python
+
+
+If you are a Windows developer and want to use PowerShell, you can use the below command.
+
+.. code-block:: powershell
+	
+	(Invoke-WebRequest -Uri https://raw.githubusercontent.com/python-poetry/poetry/master/get-poetry.py -UseBasicParsing).Content | python
+
+It will add it to path automatically. In order to make sure you have poetry installed correctly, try running:
+
+.. code-block:: bash
+
+	poetry --version
+
+
+.. note:: You may need to restart your shell in order for the changes to take effect.
+
+See the `docs on installation python poetry for more information
+<https://python-poetry.org/docs/>`_.
+
+Installing System Dependencies
+******************************
+
+Please follow the instructions under :ref:`installing-manim` to install all dependencies. Then come back to this page to continue with the installation.
```

Comment:
Is this :ref:`installing-manim` point to doc installation.rst?

Reply:
Yes, it points to the "installing manim" heading inside the `installation.rst` file.